my-code/movie-search/src/Actions/index.js

```diff
@@ -0,0 +1,25 @@
+import axios from 'axios';
+
+export const FETCH_MOVIES = 'FETCH_MOVIES';
+export const FETCH_MOVIE = 'FETCH_MOVIE';
+
+const ROOT_URL = 'http://www.omdbapi.com/?apikey=';
+const API_KEY = '5766cd6e';
+
+export function fetchMovies(searchTerm) {
+  const request = axios.get(`${ROOT_URL}${API_KEY}&s=${searchTerm}`);
```

Comment:
This is great information, Pedro! I was wondering what would be good way of catching these eventualities. I'd love to add this to what I currently have.

I have yet to try `redux-thunk`, but I think it would be a cool inclusion. I still need to grasp the idea of states, I guess.

Would it be better I learn the 'simple solution' method first so I understand what's happening in vanillaJS before using `redux-thunk`?